src/components/sessionsListItem/sessionsListItem.styles.scss

```
@@ -720,6 +727,59 @@ $session-postcode-border: #646d78;
+				color: #a5000a;
+			}
+
+			&--selected {
+				background: #a5000a;
+			}
+		}
+
+		&__caseHandoverCheckbox {
+			width: 14px;
+			height: 14px;
+			border: 2px solid currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)